lectures/time_series_with_matrices.md

```diff
@@ -64,20 +65,20 @@ y_{t} = \alpha_{0} + \alpha_{1} y_{t-1} + \alpha_{2} y_{t-2}
 ```
 
 where we assume that $y_0$ and $y_{-1}$ are given numbers
-that we take as **initial conditions**.
+that we take as *initial conditions*.
 
 In Samuelson's model, $y_t$ stood for **national income** or perhaps a different
 measure of aggregate activity called **gross domestic product** (GDP) at time $t$.
 
-Equation {eq}`tswm_1` is called a **second-order linear difference equation**.
+Equation {eq}`tswm_1` is called a **second-order linear difference equation**. It is called second order because it depends on two lags.
```

Comment:
```suggestion
Equation {eq}`tswm_1` is called a *second-order linear difference equation*. It is called second order because it depends on two lags.
```